README.md

```diff
@@ -1,2 +1,27 @@
-# xdd
-Aliases as a service
+# Hivecom Emotes (codename: xdd)
+
+Is an emote management and provider system for other hivecom projects. It allows registered users to create, edit and share emotes which can then be used in other applications which involve user input. Such as [hifriends](https://friends.hivecom.net/) or [quotes](https://quotes.hivecom.net/).
+
+The app uses authentication and permission system provided by our own authentication system [idlib](https://github.com/Mavulp/hiveID).
+
+Generally, we call the emote as `alias` because while in most cases it is an image, it is possible to also alias a text. This wider range of usage calls for a better term than `emote` as that is limited to images / gifs.
```

Comment:
In that case why is the title not "Hivecom Aliases"?

Either way you would want to write this as "We call emotes `aliases` because [...]", the "as" is not needed. Reading more of then sentence you probably want something more like "We call the values aliases and instead of emotes because even though most of the aliases will be images or gifs it is also possible to create aliases for text.".